fastai/data/external.py

```diff
@@ -10,13 +10,15 @@
 
 # Cell
 @lru_cache(maxsize=None)
-def fastai_cfg():
+def fastai_cfg(): # Config object for fastai's `config.ini`
```

Comment:
Small nit, I think we can do better. WDYT:
```python
@lru_cache(maxsize=None)
def fastai_cfg() -> Config: # A Config object for `fastai`
    "Finds a `config.ini` stemming from the env variable `FASTAI_HOME`"
```
docments should aim to *improve* the current docstring and explainability, so it wouldn't make much sense to have the docment also be 1:1 the docstring

Reply:
I agree, I have improved it a little